cfgrammar/src/lib/span.rs

```diff
@@ -103,7 +103,7 @@ impl NewlineToLineColCache {
         if line_num > self.newlines.len() {
             None
         } else {
-            Some(self.newlines[line_num - 1])
+            Some(self.newlines[line_num.saturating_sub(1)])
```

Comment:
Not sure if we need this `saturating_sub`, since this fn isn't `pub`, so we don't need to worry about being called with `0`.  I suppose this is a good reason to keep it private.

Reply:
Fixed by [a243de6](https://github.com/softdevteam/grmtools/pull/299/commits/a243de6db79874edba92b68d444e715cf2ffdafe)